packages/x-components/src/x-modules/semantic-queries/wiring.ts

```diff
@@ -8,7 +8,7 @@ import { SearchXEvents } from '../search/events.types';
 import { ExtractMutationPayload } from '../../store/store.types';
 
 /**
- * `semanticQueries` {@link XModuleName name}.
+ * `semanticQueries` {@link XModuleName | name}.
```

Comment:
Hi!
Shouldn't we also update this module `extraParams` when `QueryPreviewUnselected` is emitted??